packages/shared/prisma/migrations/20240604133340_backfill_manual_scores/migration.sql

```diff
@@ -0,0 +1,50 @@
+-- Create manual-score config for any project with manual scores and link config_id to scores
+BEGIN;
+WITH project_configs AS (
```

Comment:
I think we could have found a simpler way to move this data, which might be less elegant and produces some garbage data but cleans up. As this would happen in a transaction, this would be save anyways.